src/components/Details.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable */
+import './style/Details.css';
```

Comment:
- [ ] Kindly enable the linter in this file. Linter enforces quality code standards and therefore should not be dissabled.